fseai/fseai_models.py

```diff
@@ -0,0 +1,54 @@
+"""
+Deep Learning models in fseai
+"""
+import torch.nn as nn
+
+
+class FCmodel(nn.Module):
+    def __init__(self, input_size, hidden_size, output_size):
+        super(FCmodel, self).__init__()
+
+        # Inputs to hidden layer linear transformation
+        self.layer1 = nn.Linear(input_size, hidden_size[0])
+        # Hidden layer 1 to HL2 linear transformation
+        self.layer2 = nn.Linear(hidden_size[0], hidden_size[1])
+        # HL2 to output linear transformation
+        self.layer3 = nn.Linear(hidden_size[1], output_size)
+
+        # Define relu activation and LogSoftmax output
+        self.relu = nn.ReLU()
+        self.LogSoftmax = nn.LogSoftmax(dim=1)
```

Comment:
Objects are normally not in pascal or camel cases in python. `self.softmax` should do here